setup.py

```diff
@@ -10,7 +10,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # http://packaging.python.org/en/latest/tutorial.html#version
-    version='0.0.8',
+    version='0.1.0',
```

Comment:
At this time this is a full breaking change that is 2.9 py3 only at this moment. 
If you only want to support 2.9+ then at least allow it to boot and run in 'limp' mode on 2.8.

Reply:
reverted, you can raise the version whenever you want